bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/TeaserImpl.java

```diff
@@ -44,10 +46,30 @@
     @ScriptVariable
     protected Page currentPage;
 
+    /**
+     * The current component.
+     */
+    @ScriptVariable
+    private Component component;
+
+    @PostConstruct
+    protected void initModel() {
+
+        super.initModel();
+
+        if ((super.getTitle()!=null && !super.getTitle().isEmpty()) || super.isActionsEnabled()) {
+            super.hiddenImageResourceProperties.add(Link.PN_LINK_URL);
+        }
+
+        if (hasImage()) {
+            super.setImageResource(component, request.getResource(), super.hiddenImageResourceProperties, null);
+        }
+    }
+
     @Override
     @Nullable
     public Link getLink() {
-        return link.orElse(null);
+            return link.orElse(null);
```

Comment:
Is this indent needed?

Reply:
Are you referring to the entire function, if is needed or not?

Reply:
no, just the space before the `return` command.

Reply:
I left there by mistake. I will delete it.

Reply:
done